src/main/views/privacy-policy.njk

```diff
@@ -0,0 +1,125 @@
+{% extends "layout.njk" %}
+{% from "support.njk" import supportWidget %}
+{% set heading = t('Privacy policy') %}
+
+{% set title = t('Money claim') %}
+
+{% block content %}
+  <div class="grid-row">
+    <div class="column-two-thirds">
+      <p>{{ t('When you use the money claims service we will ask you for some personal data.' +
+        'This privacy policy explains why we collect your personal data, what we do with it and your rights.') }}
+      </p>
+      <p>
+        {{ t('The money claim service is managed by Her Majesty’s Courts and Tribunals Service and '
+        + 'will be referred to as we from now on.') }}
+      </p>
+
+      <p> {{ t('More information about how to use the service is in the '
+        +'<a href="{{  termsAndConditions }}">terms and conditions</a>.',
+        { termsAndConditions: '/terms-and-conditions' }) | safe }}
+      </p>
+
+      <h2 class="heading-medium">{{ t('Why we need your personal data') }}</h2>
+
+      <p>
+        {{ t('We collect your personal data to process your application and to meet legal requirements. '
+        + 'We also collect data to improve the service.') }}
+      </p>
+
+      <p>
+        {{ t('The personal data we need to collect includes:') }}
+      </p>
+
+      <ul class="list list-bullet">
+        <li>{{ t('your name, address and contact details') }}</li>
+        <li>{{ t('information that tells us if you open an email from us, or click on a link in an email') }}</li>
+        <li>{{ t('information about how you use this service, like your IP address and the web browser you use. '
+          + 'We do this by using <a href="{{ cookieUrl }}">cookies</a>.',
+          { cookieUrl: '/cookies'}) | safe }}
+        </li>
+      </uL>
+
+      <h2 class="heading-medium">{{ t('What we do with your data') }}</h2>
+
+      <p>
+        {{ t('HMCTS is an executive agency of the Ministry of Justice and their '
+        + '<a href="{{ personalInformationCharter }}">personal information charter</a> explains how we process data.',
+        { personalInformationCharter: 'https://www.gov.uk/government/organisations/home-office/about/personal-information-charter'}) | safe }}
+      </p>
+
+      <p>
+        {{ t('Your personal data is processed by our staff in the UK, and the data is stored in the UK.') }}
+      </p>
+      <p>
+        {{ t('We use GOV.UK Notify to send emails.
+        These are processed within the EEA until the point where emails are handed over to your email provider.') }}
```

Comment:
That will no translate well into other languages. The space between `emails.` and `These` would need to reflected in *.po file to make it work which we do not want tot have. If you breaking lines please do so by having `' + '` in between.

Reply:
👍